com.microsoft.mrtk.spatialmanipulation/Solvers/SolverHandler.cs

```diff
@@ -619,7 +617,7 @@ private bool IsHandTracked(Handedness hand)
         /// </returns>
         public static bool IsValidHandedness(Handedness hand)
         {
-            return hand <= Handedness.Both;
+            return hand.IsMatch(Handedness.Both);
```

Comment:
is None an invalid Handedness?

Reply:
None == 0

IsMatch only returns true for non-zeros

        public static bool IsMatch(this Handedness current, Handedness compare)
        {
            return (current & compare) != 0;
        }

Reply:
yes, I think the previous logic would return true for None, since 0000 < 0011 and now this logic returns false for None

Reply:
so just making sure the change is intentional